bubblewrap.c

```diff
@@ -368,7 +384,17 @@ monitor_child (int event_fd, pid_t child_pid)
           if (s == -1 && errno != EINTR && errno != EAGAIN)
             die_with_error ("read eventfd");
           else if (s == 8)
-            exit ((int) val - 1);
+            {
+              exitc = (int) val - 1;
+              if (info_fd != -1)
+                {
+                  cleanup_free char *output = xasprintf ("{\n    \"exit-code\": %i\n}\n",
+                                                         exitc);
+                  dump_info (info_fd, output);
```

Comment:
this is a change in the info-fd protocol now.  Older users didn't know they had to wait for another message and in general, I think it is fine for the users of info-fd to not wait for the exit-code if they don't need it.

In other words, the failure in `dump_info` should not cause `die_with_error`

Reply:
Should be addressed now, I changed dump_info so it optionally doesn't error